zerog/handlers/uuid.py

```diff
@@ -81,3 +81,17 @@ def do_get(self, job):
         self.complete(200, output=json.dumps(
             job.info(), indent=4, allow_nan=False)
         )
+
+
+class DumpHandler(UuidHandler):
+    def do_get(self, job):
+        jobDump = job.dump()
+        LOG_FIELDS = ['events', 'warnings', 'errors']
+        showLogs = self.get_argument('showLogs', 'false', True)
```

Comment:
Once this has been used a bit, we should keep track of whether people usually want to include the log fields or not. It's not obvious to me that this should default to False, but it's not obvious it should default to True, either.